tests/layer_validation_tests.h

```diff
@@ -270,6 +270,24 @@ class VkLayerTest : public VkRenderFramework {
         return GetPhysicalDeviceProperties2(props2);
     }
 
+    template <typename Proc>
+    const Proc GetInstanceProcAddr(const char *proc_name, bool assert_proc = true) {
```

Comment:
Use `[[nodiscard]]` for these new helper functions

```suggestion
     [[nodiscard]]  const Proc GetInstanceProcAddr(const char *proc_name, bool assert_proc = true) {
```